src/test/java/util/xcmailr/AbstractTest.java

```diff
@@ -72,25 +37,35 @@ private static void deleteTempFile(File tempFile)
             {
                 Files.delete(tempFile.toPath());
             }
-            catch (Exception e)
+            catch (final Exception e)
             {
-                System.out.println(MessageFormat.format("Couldn''t delete temporary file: ''{0}'' caused by {1}",
-                                                        tempFile.getAbsolutePath(), e));
+                // double apostrophe needed, otherwise MessageFormat.format() won't work
+                System.out.println(MessageFormat.format(
+                                                        "Coundn''t delete temporary file: ''{0}'' caused by {1}",
+                                                        tempFile.getAbsoluteFile(), e));
             }
         }
     }
 
+    protected void writeProperty(String propertyName, String propertyValue)
+    {
+        properties2.put(propertyName, propertyValue);
+        writeMapToPropertiesFile(properties2, tempConfigFile2);
+        ConfigFactory.setProperty("xcmailr.temporaryConfigFile", "file:" + fileLocation);
+        XcMailrApi.resetConfigurationsForThread();
```

Comment:
This isn't needed if your properties stay the same in each test class which is definitely possible.